ecosystem/add-ons-intro.md

```diff
@@ -0,0 +1,2 @@
+Add-ons provide support for additional use cases and scenarios for your
```

Comment:
Maybe "provide additional utility and expand on what Trino has to offer" or something like that? "additional use cases and scenarios" doesn't really mean much.